### PR TITLE
Add support for compy

### PR DESCRIPTION
**Pull request by milenaveneziani**
This is mostly to add support for compy.

While I am at it, I also:
- updated the way we activate e3sm-unified (pointing to the py2.7 version because our python scripts in a-prime are not compatible with python 3)
- removed edison and wolf
- updated small things like the name of the E3SM repository

### CODE REVIEW

**Comment by milenaveneziani** [comment]
This needs to be tested. I will use a recent coupled simulation on compy, probably the one Luke ran for testing the depth-dependent GM.

**Comment by milenaveneziani** [comment]
Worked like a charm. Results can be seen here:
https://portal.nersc.gov/project/acme/milena/20190723.testLuke2.A_WCYCL1850S.ne30_oECv3.compy_years41-50_vs_obs/

@xylar: if you could take a quick look, especially at the bits where we load e3sm-unified, and approve this, I'll merge soon after.

**Comment by salilmahajan** [comment]
Yay! Thanks, Milena!

________________________________

**Comment by xylar** [comment]
Great! I’ll take a look on Monday.

**Review by xylar — APPROVED** [review]
It would be worth considering updating the version of MPAS-Analysis that A-Prime uses to the latest, which would require updating https://github.com/E3SM-Project/a-prime/blob/master/python/setup_ocnice_config.py.  That should be a separate PR.

**Comment by xylar** [comment]
Could you rebase onto #42 or include the other change from that PR?

**Comment by xylar** [comment]
A couple more notes.  

* I will make an update to e3sm-unified soon but I will not do anymore py2.7 updates so e3sm-unified so the current version is the last unless/until you to update a-prime to py3
* I would strongly suggest doing a separate PR to remove MPAS-Analysis as a submodule and instead use MPAS-Analysis from the E3SM-Unified you're loading.  This might mean you really need to be loading a specific E3SM-Unified rather than the "latest" unless you know A-Prime will remain compatible with the latest MPAS-Analysis.  The reason to use MPAS-Analysis from E3SM-Unified is that this is what users might expect, and it might be very confusing that an older version is downloaded and used instead.

File: run_aprime.bash [hunk 1]
```diff
@@ -169,18 +167,20 @@ elif [ ${HOSTNAME:0:5} == "aims4" ]; then
   export machname="aims4"
 elif [ ${HOSTNAME:0:5} == "acme1" ]; then
   export machname="acme1"
-elif [ ${HOSTNAME:0:4} == "wolf" ] || [ ${HOSTNAME:0:7} == "grizzly" ]; then
+elif [ ${HOSTNAME:0:7} == "grizzly" ]; then
```
[the remark this answers is not in the document]
Comment (milenaveneziani) [reply run_aprime.bash]: I can surely add badger here

**Comment by milenaveneziani** [comment]
@xylar: about your last two notes. Those changes would be really substantial for a-prime. I think they would make sense if, within e3sm, we agree that they are worthwhile, also considering the future of a-prime. This package has been heavily underused over the past year, but it is possible that its usefulness will come in handy again during v2 tuning, when quick evaluations of the coupled system will be necessary again. Perhaps we can discuss this at a next water cycle phone call.

**Comment by milenaveneziani** [comment]
And thanks @xylar for the weekend review :)

**Pull request merged**